scripts/revcheck.php

```diff
@@ -616,9 +616,20 @@ function print_html_files( $enFiles , $trFiles , $lang )
             case FileStatusEnum::TranslatedCritial: $bg = 'bgred'   ; break;
             default:                                $bg = 'bggray'  ; break;
         }
-        $d1 = "https://git.php.net/?p=doc/en.git;a=blobdiff_plain;f=$key;hb={$en->hash};hpb={$tr->hash};"; // text
-        $d2 = "https://git.php.net/?p=doc/en.git;a=blobdiff;f=$key;hb={$en->hash};hpb={$tr->hash};";       // html
-        $nm = "<a href='$d1'>{$en->name}</a> <a href='$d2'>(h)</a>";
+        // GitHub web diff -- May not work with very old commits
+        $kh = hash( 'sha256' , $key );
+        $d1 = "https://github.com/php/doc-en/compare/{$tr->hash}..{$en->hash}#diff-{$kh}";
+        // Local git diff  -- Always work
+        $d2 = "(cd en; git diff {$tr->hash} {$en->hash} -- $key)";
+        $d2 = htmlspecialchars( $d2 );
```

Comment:
htmlspecialchars is the wrong method to use with `data-clipboard-text='{$d2}'` - I assume the use of single quotes in the original code is just convenience over `\"`

```
php > echo htmlspecialchars("'");
'
php > echo htmlspecialchars("\"");
&quot;
```

Reply:
Thanks, and good catch. Added ENT_QUOTES ensure all quotes are always escaped.